comp/logs/agent/config/integration_config.go

```diff
@@ -82,6 +82,14 @@ type LogsConfig struct {
 	Tags            []string
 	ProcessingRules []*ProcessingRule `mapstructure:"log_processing_rules" json:"log_processing_rules"`
 
+	// This is tightly linked to how messages are transmitted through the pipeline.
+	// If set to true, tailers using structured message (journald, windowsevents)
+	// will fall back to previous behavior of sending the whole message (e.g. JSON
+	// for journald) for post-processing. Which is the behavior of Agent < 7.50.0.
+	// Otherwise, the message content is extracted from the structured message and
+	// only this part is post-processed and sent to the intake.
+	V1Behavior bool `mapstructure:"v1_behavior" json:"v1_behavior"`
```

Comment:
Perhapse this should be more specific? ex: `v1_processing_rules_behavior` or `v1_processing_behavior`
I can see a future where we make other beahvior changes and could possibly need similar (but functionally different) flags.

Reply:
Yeah, I absolutely have to rename it to something better, adding this to the TODO list for sure.
`v1_processing_behavior` SGTM, I'll go with this in next commit but will still spend some time putting more thoughts into this (On top of that, I'll have to write documentation about this, it might help finding the best scenario)